feeds/wlan-ap/opensync/src/vendor/tip/src/lib/target/inc/target_tip.h

```diff
@@ -52,6 +52,7 @@ typedef struct
 	uint32_t chan_tx;
 	uint32_t chan_noise;
 	uint32_t duration_ms;
+	uint32_t chan_use;
```

Comment:
'chan_use' is not a great name.  'chan_in_use' or 'active_chan' would have been more readable.

Reply:
done